README.md

```diff
@@ -21,8 +21,8 @@ If your team wants to use this, follow [Getting started](#getting-started).
 - Test summary with errors from all test runs
 - Configure how debug logs get parsed and interpreted
 - Fancy progress bar
-- Uses qemu snapshots
-- Individual test configuration via a test headers (timeout, expected exit codes)
+- Uses qemu snapshots for faster bootup
```

Comment:
I would suggest `- Uses qemu snapshots to avoid booting multiple times` or removing this point.

Reply:
Good point!